daemon/logic/utils.go

```diff
@@ -368,38 +368,55 @@ func createKeyringMemberships(ctx context.Context, c *crypto.Engine, client *reg
 	return v1members, v2members, nil
 }
 
-// fetchKeyPairs fetches the user's signing and encryption keypairs from the
-// registry for the given org id.
-func fetchKeyPairs(ctx context.Context, client *registry.Client,
-	orgID *identity.ID) (*identity.ID, *identity.ID, *crypto.KeyPairs, error) {
+// fetchRegistryKeyPairs fetches the user's signing and encryption keypairs
+// from the registry for the given org id.
+// It returns an error if the keypairs cannot be fetched from the registry,
+// or if an invalid keypair type is seen.
+// It returns nil for keypair types that are not found.
+func fetchRegistryKeyPairs(ctx context.Context, client *registry.Client,
```

Comment:
This assumes a user only ever has at most one keypair for an organization.

Is there any situation where they might have two?